### ISSUE TITLE
Improve BezPath docs

### ISSUE THREAD

**Issue by rylev** [linebender/kurbo#10]
The docs for `BezPath`'s methods seem a bit too literal. For instance, `curveto` says "Push a curveto element onto the path." which is technically correct, but something like "appends a cubic Bézier curve to the path" might be more helpful. I found Apple's UIKit docs for this pretty nice: https://developer.apple.com/documentation/uikit/uibezierpath/1624357-addcurve

**Comment by raphlinus** [comment]
Yes, it's pretty basic right now and could definitely be improved. One question is whether the primary point of documentation for that should be the `curveto` method or the `PathEl::Curveto` enum. I think I'd lean toward the former, as it's more similar to the PostScript original model, and more likely to be the point of contact.

Another question: is it practical to include SVG in kurbo rustdoc? Math? I do both in my blog, and I'm ok with pushing the boundaries a bit here. If we can't do it in rustdoc, we should have a canonical place (maybe a docs subdir) where it is enabled.

**Comment by jrus** [comment]
I don’t know whether it’s directly relevant to this conversation, but personally I find it a lot nicer to directly manipulate some data structure listing control points or the like, and then pass it into a function that can draw the whole shape, rather than repeatedly calling .curveTo, .lineTo, etc.

Having the data stored as data instead of a black box that gets appended to bit by bit with a procedural API is more flexible to interact with. But maybe less familiar to people used to existing 1970s style procedural APIs.
